pkg/cmd/kafka/acl/list/list.go

```diff
@@ -116,6 +121,9 @@ func NewListACLCommand(f *factory.Factory) *cobra.Command {
 	flags.AddUser(&userID)
 	flags.AddServiceAccount(&serviceAccount)
 	flags.AddAllAccounts(&allAccounts)
+	flags.AddCluster(&opts.cluster)
+	flags.AddTopic(&opts.topic)
+	flags.AddConsumerGroup(&opts.group)
```

Comment:
We need to add dynamic completions for these.

Reply:
Will it be good to add an additional field `factory` in the [flagSet struct](https://github.com/redhat-developer/app-services-cli/blob/eb27c0f575a816945cd75d4f9ce777c2dea28fee/pkg/cmd/kafka/acl/flagutil/flagset.go#L25)? That will make it easier to reuse completions for topic and group from cmdutils?

Reply:
Yeah, that would be good. You can remove the localizer and conn fields so and just use this :)

I'd recommend refactoring these completions APIs also to look like this one: https://github.com/redhat-developer/app-services-cli/blob/8edfc44c74d1b21974ad97ae0d5c096c61818587/pkg/kafka/cmdutil/util.go#L11

They also probably should not be in a generic cmdutil package since they are kafka specific (I did that 😂)